app/scripts/components/common/featured-slider-section.tsx

```diff
@@ -140,9 +131,9 @@ function FeaturedSliderSection(props: FeaturedSliderSectionProps) {
                           )}
                         </CardMeta>
                       }
-                      description={cardDescription}
-                      imgSrc={cardMedia?.src}
-                      imgAlt={cardMedia?.alt}
+                      description={getDescription(d)}
+                      imgSrc={getMediaProperty(undefined, d, 'src')}
```

Comment:
I'd prefer `null` instead of `undefined` in this case, to mark it as intentional. 
We don't have to change it now, since we use `undefined` in several places throughout the code already, but maybe consider it for the future?

Reply:
Noted, will consider switching to `null` next time 👍